src/video-grid/VideoTile.tsx

```diff
@@ -108,13 +114,22 @@ export const VideoTile = forwardRef<HTMLDivElement, Props>(
       }
     }, [member, setDisplayName]);
 
-    const muted =
-      useMediaTrack(
-        content === TileContent.UserMedia
-          ? Track.Source.Microphone
-          : Track.Source.ScreenShareAudio,
-        sfuParticipant,
-      ).isMuted !== false;
+    const audioInfo = useMediaTrack(
+      content === TileContent.UserMedia
+        ? Track.Source.Microphone
+        : Track.Source.ScreenShareAudio,
+      sfuParticipant,
+    );
+    const videoInfo = useMediaTrack(
+      content === TileContent.UserMedia
+        ? Track.Source.Camera
+        : Track.Source.ScreenShare,
+      sfuParticipant,
+    );
+    const muted = audioInfo.isMuted !== false;
+    const encrypted =
+      audioInfo.publication?.isEncrypted !== false &&
+      videoInfo.publication?.isEncrypted !== false;
```

Comment:
Perhaps a matter of style/preference, but I'd prefer to explicitly handle `undefined` instead of performing a comparison that would "convert" it into `boolean`.

```suggestion
    const muted = audioInfo.isMuted ?? false;
    const encrypted =
      audioInfo.publication?.isEncrypted ?? false &&
      videoInfo.publication?.isEncrypted ?? false;
```